c-api/lib.rs

```diff
@@ -592,14 +604,24 @@ pub extern "C" fn resvg_render(
     let pixmap: &mut [u8] = unsafe { std::slice::from_raw_parts_mut(pixmap as *mut u8, pixmap_len) };
     let pixmap = tiny_skia::PixmapMut::from_bytes(pixmap, width, height).unwrap();
 
-    resvg::render(&tree.0, fit_to.to_usvg(), tiny_skia::Transform::default(), pixmap).unwrap()
+    let skia_xf = tiny_skia::Transform::from_row(
```

Comment:
Prefer:

```rust
impl resvg_transform {
    #[inline]
    fn to_tiny_skia(&self) -> tiny_skia::Transform {
        tiny_skia::Transform::from_row(
            self.a as f32, 
            self.b as f32, 
            self.c as f32, 
            self.d as f32, 
            self.e as f32, 
            self.f as f32,
        );
    }
}
```

and then `transform.to_to_tiny_skia()`, just like with `resvg_fit_to`